### PR TITLE
Remove the queryparam ?beta=true in perspectives app

### PR DESCRIPTION
**Pull request by pallavi2209**

### CODE REVIEW

File: tests/view/perspectives/app.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 'use strict';
 
-const expect = require('chai').expect;
-const utils = require(
-  '../../../view/perspective/utils'
-);
+const chai = require('chai');
+const expect = chai.expect;
+import { getFilterQuery,
+  getTagsFromArrays } from '../../../view/newPerspective/utils.js';
```
Comment (iamigo) [inline tests/view/perspectives/app.js]: "newPerspective" should be "perspective" here, right?

File: tests/view/perspectives/createPerspective.js [hunk 2]
```diff
@@ -13,7 +13,7 @@
 import { expect } from 'chai';
 import React from 'react';
 import sinon from 'sinon';
-import CreatePerspective from '../../../view/perspective/CreatePerspective';
+import CreatePerspective from '../../../view/newPerspective/CreatePerspective';
```
Comment (iamigo) [inline tests/view/perspectives/createPerspective.js]: "newPerspective" should be "perspective" here, right?

File: tests/view/perspectives/eventsQueue.js [hunk 3]
```diff
@@ -13,7 +13,7 @@
 
 const expect = require('chai').expect;
 const eventsQueue = require(
-  '../../../view/perspective/eventsQueue'
+  '../../../view/newPerspective/eventsQueue'
```
Comment (iamigo) [inline tests/view/perspectives/eventsQueue.js]: "newPerspective" should be "perspective" here, right?

File: view/perspective/perspective.pug [hunk 4]
```diff
@@ -34,7 +34,7 @@ html(lang = 'en')
   script
     if queryParams
       | var queryParams = !{queryParams}
-  script(src='/static/perspective/app.js')
+  script(src='/static/newPerspective/app.js')
```
Comment (iamigo) [inline view/perspective/perspective.pug]: "newPerspective" should be "perspective" here, right?

**Comment by pallavi2209** [comment]
@iamigo I will deploy this and test that realtime events works and then I will let you when this is ready for review :)

**Comment by pallavi2209** [comment]
@iamigo @annyhe PR ready for review. I verified that bulkupsert and realtime events work. The changes are deployed here: https://refocus-devbox.herokuapp.com

**Pull request merged**